### ISSUE TITLE
Extend EE tests to exercise deployed app on stage and run

### ISSUE THREAD

**Issue by ldimaggi** [fabric8io/fabric8-test#268]

**Comment by ppitonak** [comment]
@ldimaggi can you please more information? Do you want to test the functionality of application itself?

**Comment by ldimaggi** [comment]
I don't think that we have to test the actual operation of the deployed quickstart at its endpoint. That testing should instead be tested by the quickstart tests (we're tracking those tests in separate issues).

What I think we should verify in the E2E tests is that verify that the deployment to stage and/or run was successful. There have been instances of deployments being reported as being successful int he OOIO UI when in fact, the deployments were not successful.

For example, see issue - https://github.com/openshiftio/openshift.io/issues/2629 - I would like to see the E2E tests verify that the deployments were successful. Thx!
